build-system/server.js

```diff
@@ -346,16 +346,19 @@ function liveListTombstone(liveList) {
   }
 }
 
+
+// Value is inclusive of both min and max values.
+function range(min, max) {
```

Comment:
Yea, I'm not sure what the use is creating the `values` array in the first place.